pkg/cmd/server/api/types.go

```diff
@@ -325,6 +325,14 @@ type AuditConfig struct {
 	// If this flag is set, audit log will be printed in the logs.
 	// The logs contains, method, user and a requested URL.
 	Enabled bool
+	// All requests coming to the apiserver will be logged to this file.
+	AuditFilePath string
```

Comment:
I don't see validation on these fields and I don't see this path listed in the list of paths for resolution.